examples/ui-demo/src/components/modals/Erc20/Erc20Modal.tsx

```diff
@@ -191,12 +190,11 @@ export function Erc20Modal({
   const buyNftButtonEnabled =
     readyToBuyNft && !isMintingNft && !isLoadingNftClient;
 
-  let bagIconColor = "#CBD5E1";
-  if (mintNftTxHash) {
-    bagIconColor = "#475569";
-  } else if (buyNftButtonEnabled) {
-    bagIconColor = "white";
-  }
+  let bagIconColor = mintNftTxHash
```

Comment:
✨ this can be `const` instead of `let` now

Reply:
ahh sorry was making the updates in between meetings and making dumb mistakes.  I feel like my linter should have caught this though